packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -7,7 +7,7 @@ import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
 import normalizeWheel from '@conversionxl/normalize-wheel';
 import cxlAppLayoutStyles from '../styles/cxl-app-layout-css.js';
 import cxlAppLayoutGlobalStyles from '../styles/global/cxl-app-layout-css.js';
-import '@vaadin/vaadin-button';
+import '@vaadin/button';
 import '@vaadin/vaadin-context-menu/src/vaadin-device-detector.js';
```

Comment:
This should refactor to `MediaQueryController`.

Reply:
I'm down for moving off of device detector. I often use this utility:

```
export const observe = (mediaQueryString, callback) => {
    const observer = window.matchMedia(mediaQueryString);
    const matches = (mediaQueryList) => callback(mediaQueryList.matches);
    observer.addListener(matches);
    matches(observer);
};
```

Hooking on to Vaadin's code would be great for maintainability though, I will get that done in the next few days.

Reply:
Shouldn't we handle Media Query upgrade in separate PR?